### DISCUSSION TITLE
Docker compose up no longer works.

### DISCUSSION THREAD

**Discussion by hopeseekr** [AllYourBot/hostedgpt#358] [General]
My first time trying to install the service.

The `docker compose up` directions are not working on new installs as of 2024-05-19. While I am quite experienced in Docker and other languages (C++, C#, PHP, JavaScript), I have zero experience with Ruby nor Tailwind and am unable to troubleshoot this.

----

When you do `docker compose up` now you get hit with:

> Cannot render console from 172.25.0.1! Allowed networks: 192.168.0.0/192.168.255.255

So I went in and edited `config/environments/development.rb` and changed line 79 from

```
   config.web_console.permissions = "192.168.0.0/16"
```
to
```
   config.web_console.permissions = "172.0.0.0/16"
```

so that it, you know, runs on Docker networks...

But then I get hit with a big-ole **Sprockets::Rails::Helper::AssetNotFound in Sessions#new**

**The asset "tailwind.css" is not present in the asset pipeline.**

error.
![image](https://github.com/allyourbot/hostedgpt/assets/1125541/b7c131ff-c718-42b6-a23c-f524f3d998b9)

Also, I want to run this via docker in production. How do I change the environment???

It seems you left stuff out of the README and/or the docker compose setup has bitrotted in just 3 months to the point of not being able to be setup on the first run.

**Comment by keithschacht** [comment]
I saw a different comment first and already converted this one over to an issue: https://github.com/allyourbot/hostedgpt/issues/359 
Take a look when you're able to.
